tests/psa-client-server/psasim/src/aut_psa_aead_encrypt_decrypt.c

```diff
@@ -4,6 +4,7 @@
  */
 
 #include "psa/crypto.h"
+#include "../tf-psa-crypto/core/common.h"
```

Comment:
Escaping to a parent tree like this means that we can't easily change the structure of the build tree with respect to the source tree. In particular this may not work with an out-of-tree build. I'm ok with this as a hack for now, because doing it properly looks complicated. But please add a link to the corresponding issue: https://github.com/Mbed-TLS/mbedtls/issues/10223